presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitSource.java

```diff
@@ -328,7 +329,7 @@ public synchronized ListenableFuture<List<ConnectorSplit>> borrowBatchAsync(Opti
                     {
                         checkArgument(bucketNumber.isPresent(), "bucketNumber must be present");
                         if (!allSplitLoaded.isDone()) {
-                            return allSplitLoaded.transform(ignored -> ImmutableList.of(), executor);
+                            return FluentFuture.from(allSplitLoaded).transform(ignored -> ImmutableList.of(), executor);
```

Comment:
How is this change related?

Reply:
Fixes a compilation error due to this change introduced in Guava 27.0: [AbstractFuture doesn't expose FluentFuture APIs anymore](https://github.com/google/guava/commit/0f8d3608af401d9090642da2b88f9a8b5999ff1f)